utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -349,6 +349,7 @@ Describe 'Readme tests' -Tag Readme {
                 }
 
                 $readmeCategoryColumns = ($readMeContent[$tableStartIndex] -split '\|') | ForEach-Object { $_.Trim() } | Where-Object { -not [String]::IsNullOrEmpty($_) }
+
```

Comment:
👀

Reply:
Removed - though this seems very excessive :D